corehq/apps/app_manager/static/app_manager/js/releases/releases.js

```diff
@@ -252,7 +252,9 @@ hqDefine('app_manager/js/releases/releases', function () {
 
         self.download_modal = $(self.options.download_modal_id);
         self.async_downloader = asyncDownloader(self.download_modal);
-
+        self.savedApps.subscribe(() => {
+            self.options.appReleaseLogs.goToPage(1);
+        });
```

Comment:
@pxwxnvermx what is this needed for?

Reply:
this refreshes the app release logs table whenever an application is created, updated or deleted.

Reply:
Oh great. And the hook in the ajax call is for released/In-test updates. Cool!